### PR TITLE
Add christmas card example with docs

### PR DESCRIPTION
**Pull request by mountainflo**
Add complete example for the christmas card including:

* short description of the task with helpful links
* HTML and CSS code
* backgound image
* added a link to the task in the main project README

### CODE REVIEW

**Comment by mountainflo** [comment]
@CraftyKoala Here is a first example for the participants, which they can take as a suggestion if they want. And which they can of course extend as they wish.
Maybe you can think of a few more extensions for the Christmas card, which we can note down as a backup in case the participants themselves don't have any more ideas.

File: christmas-card/README.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+
+## Ziel dieses Projekts
+
+Für deine Weihnachtskarte kannst du HTML und CSS verwenden. Versuche auch Links in die Weihnachtskarte einzubauen.
+
+Hier eine Übersicht, was wir bereits mit HTML alles gemacht haben:
+* Texte mit `<p>`, `<h1>`, `<h2>`, `<h3>`
+* Container (Boxen, Schachteln) mit `<div>`
+* Links mit `<a href="hier-steht-der-link"></a>`
+* Bilder mit `<img src="name-der-bilddatei.jpg" alt="Alternativer Text bei Fehlern" />`
```
Comment (CraftyKoala) [inline christmas-card/README.md]: Worauf verweist du in diesem Block ?

File: christmas-card/README.md [hunk 2]
```diff
@@ -1 +1,82 @@
...
+    background-repeat: no-repeat;                   /* wiederholung des Bildes */
+    background-position: center center;             /* horizontale und vertikale Position des Bildes auf der Webseite */
+    background-attachment: fixed;                   /* das Bild scrollt nicht mit dem Inhalt der Seite mit */
+    background-size: cover;                         /* Bild wird vergrößert, bzw. verkleinert, um die gesamte Seite auszufüllen */
+}
+```
+
+## Weihnachtskarte mit Freunden teilen
+
+In `repl.it` hast du die Möglichkeit deine Webseite auch in einem separaten Tab zu öffnen.
```
Comment (CraftyKoala) [inline christmas-card/README.md:79]: Button oben rechts

File: christmas-card/README.md [hunk 3]
```diff
@@ -1 +1,82 @@
...
+Die URL des neuen Tab kannst du kopieren und an Freunde und Familie verschicken.
+
+**Wichtig:** Damit deine Webseite auch von deinen Freunden angeschaut werden kann, musst du die Sichtbarkeit deines `repls` auf öffentlich stellen.
```
Comment (CraftyKoala) [inline christmas-card/README.md]: Vorschlag :  ...öffentlich (`public`) stellen.

**Comment by CraftyKoala** [comment]
> @CraftyKoala Here is a first example for the participants, which they can take as a suggestion if they want. And which they can of course extend as they wish.
> Maybe you can think of a few more extensions for the Christmas card, which we can note down as a backup in case the participants themselves don't have any more ideas.

* They could also include icons to the card: https://www.w3schools.com/icons/ (maybe not font awesome 5)
* use some gifs

File: christmas-card/README.md (hunk 3 above)
Comment (mountainflo) [reply christmas-card/README.md]: ack

File: christmas-card/README.md (hunk 2 above)
Comment (mountainflo) [reply christmas-card/README.md:79]: ack

File: christmas-card/README.md (hunk 1 above)
Comment (mountainflo) [reply christmas-card/README.md]: hab es nochmal etwas umformuliert

**Comment by mountainflo** [comment]
> > @CraftyKoala Here is a first example for the participants, which they can take as a suggestion if they want. And which they can of course extend as they wish.
> > Maybe you can think of a few more extensions for the Christmas card, which we can note down as a backup in case the participants themselves don't have any more ideas.
> 
> * They could also include icons to the card: https://www.w3schools.com/icons/ (maybe not font awesome 5)
> * use some gifs

Danke für die tollen Ideen für mögliche Erweiterungen

**Pull request merged**